From: Robert C Jennings <rcj4747@gmail.com>
Date: Mon, 1 Feb 2016 21:14:59 -0600
Subject: Make destination directory prior to installation

---
 install.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/install.sh b/install.sh
index 10d1856..7fc79d5 100755
--- a/install.sh
+++ b/install.sh
@@ -82,6 +82,7 @@ install_tools() {
   done
 
   info "Copying native_manpath.sh to $INSTALL_DIR/etc/profile.d/"
+  mkdir -p $INSTALL_DIR/etc/profile.d/
   cp ./src/etc/profile.d/native_manpath.sh $INSTALL_DIR/etc/profile.d/
 
   info "Copying ./src/lib/smartdc to $INSTALL_DIR/lib/"
@@ -92,6 +93,7 @@ install_debian() {
   install_tools
 
   info "Installing upstart job to $INSTALL_DIR/etc/init/"
+  mkdir -p $INSTALL_DIR/etc/init/
   cp ./src/lib/smartdc/joyent-lxtools.conf $INSTALL_DIR/etc/init/
 }
 
